FOLLOW-UP.md

```diff
@@ -4,18 +4,16 @@
 
 ### Q) What is the command to start the server?
```

Comment:
```suggestion
### Q) What's the command to start the local dev environment?
```

I've been meaning to change this for ages, could we make it something that covers the whole test? I've included a suggestion but we don't need to make it exactly that, just something along the same lines.